purview/api/v1.py

```diff
@@ -104,3 +104,51 @@ def get_resource_story(resource, uid):
     results['meta']['related_nodes'].update(get_corelated_nodes(results_unordered))
 
     return jsonify(results)
+
+
+@api_v1.route('/allstories/<resource>/<uid>')
+def get_resource_all_stories(resource, uid):
+    """
+    Get all unique stories of a resource from Neo4j.
+
+    :param str resource: a resource name that maps to a neomodel class
+    :param str uid: the value of the UniqueIdProperty to query with
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raises NotFound: if the item is not found
+    :raises ValidationError: if an invalid resource was requested
+    """
+    item = get_neo4j_node(resource, uid)
+    if not item:
+        raise NotFound('This item does not exist')
+
+    for _, prop_def in item.__all_properties__:
```

Comment:
This could be a separate function since `get_resource_story` has the same code as well.

Reply:
Can you suggest a nice name for this? since we already have a `create_query` function.